apps/hpc-ftsadmin/src/app/utils/fn-promises.ts

```diff
@@ -0,0 +1,91 @@
+import { categories } from '@unocha/hpc-data';
+import { Environment } from '../../environments/interface';
+import { FormObjectValue } from '@unocha/hpc-ui';
+
+/** Functions to pass to <AsyncAutocompleteSelect /> fnPromise prop */
+
+export const fnOrganizations = async (
+  query: { query: string },
+  env: Environment
+) => {
+  const response =
+    await env.model.organizations.getAutocompleteOrganizations(query);
+  const options: Array<FormObjectValue> = response.map((responseValue) => {
+    return {
+      displayLabel: `${responseValue.name} [${responseValue.abbreviation}]`,
+      value: responseValue.id,
+    };
+  });
+  return options;
+};
```

Comment:
You can return the mapped array directly, without storing it to a variable first. Also, you can simplify returns in arrow functions. For slightly less verbose code, you cal apply similar pattern across this file:
```suggestion
export const fnOrganizations = async (
  query: { query: string },
  env: Environment
) => {
  const response =
    await env.model.organizations.getAutocompleteOrganizations(query);

  return response.map((responseValue) => ({
    displayLabel: `${responseValue.name} [${responseValue.abbreviation}]`,
    value: responseValue.id,
  }));
};
```